src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigStorageLocationExtensions.cs

```diff
@@ -1,25 +1,26 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Immutable;
+using System.Collections.Generic;
 using Microsoft.CodeAnalysis.Diagnostics;
-using Roslyn.Collections.Immutable;
 
 namespace Microsoft.CodeAnalysis.Options
 {
     internal static class EditorConfigStorageLocationExtensions
     {
         public static bool TryGetOption(this IEditorConfigStorageLocation editorConfigStorageLocation, AnalyzerConfigOptions analyzerConfigOptions, Type type, out object value)
         {
-            var optionDictionary = analyzerConfigOptions.Keys.ToImmutableDictionary(
-                key => key,
-                key =>
-                {
-                    analyzerConfigOptions.TryGetValue(key, out var optionValue);
-                    return optionValue;
-                });
+            // This is a workaround until we have an API for enumeratings AnalyzerConfigOptions.
```

Comment:
Probably add a link to the issue?

Reply:
I'll open an issue to reference https://github.com/dotnet/roslyn/issues/41840